packages/craco/README.md

```diff
@@ -165,6 +165,10 @@ module.exports = {
     webpack: {
         alias: {},
         plugins: [],
+        removePlugins: {
```

Comment:
I wasn't thinking about introducing a new prop for this. My thought was to make the plugin prop smarter, by also accepting an array for replacing an existing plugin or removing one.

for exemple:

```
webpack: {
    plugins: [
       new ConfigWebpackPlugin(),
       [new HtmlWebpackPlugin(...), { action: "replace" }],
       ["ReactRefreshWebpackPlugin", { action: "remove" }]
    ]
}
```

While writing this I find it quite sloppy hehe so I am open to something else.

Maybe:

```
webpack: {
    addPlugins: [new ConfigWebpackPlugin()],
    replacePlugins: [{ name: "HtmlWebpackPlugin", new HtmlWebpackPlugin(...) }],
    removePlugins: ["ReactRefreshWebpackPlugin"]
}
```

For `replacePlugins`, the plugin name could maybe be inferred from the new instance.

I might prefer the following style:

```
webpack: {
    plugins: {
       add: [new ConfigWebpackPlugin()],
       replace: [{ name: "HtmlWebpackPlugin", new HtmlWebpackPlugin(...) }],
       remove: ["ReactRefreshWebpackPlugin"]
    }
}
```

What do you think?